### PR TITLE
[ES] Components and fuzzy solved

### PR DESCRIPTION
**Pull request by luischos**
es-06
es-07

### CODE REVIEW

**Comment by qwrtln** [comment]
po4a check

File: translations/components.tex/es.po [hunk 1]
```diff
@@ -77,65 +53,21 @@ msgid ""
...
-"\\end{minipage}\n"
-"\n"
+"  \\begin{subfigure}[t]{0.4\\linewidth}\n"
+"    \\includegraphics[width=\\linewidth]{\\images/combat_board.png}\n"
+"    \\caption{\\textbf{\\pagelink{Combat}{Tablero de Combate}}}\n"
+"  \\end{subfigure}\n"
+"  ~\n"
+"  \\begin{subfigure}[t]{0.4\\linewidth}\n"
+"    \\includegraphics[width=\\linewidth]{\\images/battlefield-alone.png}\n"
+"    \\caption{\\textbf{Expansión de Batalla} \\\\\\textit{(Campo de Batalla)}}\n"
```
Comment (qwrtln) [inline translations/components.tex/es.po:65]: I think you mixed up the two names.

File: translations/components.tex/es.po [hunk 2]
```diff
@@ -259,33 +159,43 @@ msgid ""
...
+"  \\begin{subfigure}[c]{0.12\\linewidth}\n"
 "    \\centering\n"
-"    \\includegraphics[width=0.75\\linewidth]{\\images/school_of_earth.png}\n"
-"    \\textit{\\textbf{\\textcolor{darkcandyapplered}{Escuela de la Tierra}}}\n"
-"  \\endminipage\n"
-"  \\bigbreak\n"
-"\\end{minipage}\n"
-"\n"
+"    \\includegraphics[width=5em]{\\images/build.png}\n"
+"    \\caption{\\textbf{\\pagelink{Town Actions}{Construir}} \\iftoggle{printable}{}{\\phantom{Population}}}\n"
```
Comment (qwrtln) [inline translations/components.tex/es.po:187]: Don't you want "Ficha de" here?

Comment (luischos) [reply translations/components.tex/es.po:187]: If I put "Ficha de" there, the page will disarm, I need to put short names

**Comment by luischos** [comment]
po4a commit

**Comment by luischos** [comment]
@qwrtln done!

**Review by qwrtln — CHANGES_REQUESTED** [review]
Not quite.
1. There's still a fuzzy string in Combat section. This could be due to changing the original string by accident. Please fix it.
2. Please add updated high quality images of both digital and printable versions. You can use `tools/pdf2image.sh` script for this.
3. You shouldn't rename or shorten components' names to make them fit on the page. Instead, you should work on changing the layout slightly. Please see `\iftoggle{printable}{}{}` mechanism that was added for this. In case of doubts, please reach out to me.

**Comment by DarthGandalf** [comment]
Are you sure "Expansión de Batalla" is the name of the component, and "Campo de Batalla" is which expansion it comes from, not vice versa? :)

File: translations/components.tex/es.po (hunk 1 above)
Comment (qwrtln) [reply translations/components.tex/es.po]: They are still mixed up.

**Comment by luischos** [comment]
Point 1 and 2 solved, and edited the first image with a new pdf capture (now you can see the text correctly).
What do I have to do with the 3rd point?

**Comment by luischos** [comment]
@qwrtln done!

**Comment by qwrtln** [comment]
Perfect! Now please restore the actual names of the components without skipping "Carta" or "Ficha de". Make a digital build, and show the high-res images here. Disregard the printable version or the failing build for now.

**Comment by luischos** [comment]
@qwrtln the images
es-06
es-07
es-08
es-09

**Comment by qwrtln** [comment]
Does the image show the code on github? It doesn't look so, because the workflow would fail.

**Comment by qwrtln** [comment]
In any case, this looks pretty good. I think that if you shrink the battlefield board and the dice for the printable build, everything will fit perfectly.

**Comment by qwrtln** [comment]
Oh, and battlefield obstacles too!

**Comment by qwrtln** [comment]
@luischos done, pull, build, and post new printable images here :-)

**Comment by luischos** [comment]
@qwrtln  cool :)
es-06
es-07

**Review by qwrtln — APPROVED** [review]

**Pull request merged**